packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -81,7 +81,15 @@ export interface StakePoolMetrics {
   delegators: number;
 
   /**
-   * Rewards Annual Percentage Yield (APY).
+   * The Annual Percentage Yield (APY) from delegators prospective in the given period.
```

Comment:
```suggestion
   * The Annual Percentage Yield (APY) from a delegator's perspective in the given period.
```

Reply:
e5912e690 should solve this

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/583#discussion_r1096002250

Reply:
sorry...
a75ee9bf6 should solve this